bin/blanksky

```diff
@@ -162,28 +172,35 @@ def e_norm(evtfile,bkgfile,chip,bkgparams,instrument):
     ccd = f"[{det}={chip}]"
 
     dmlist.punlearn()
-    dmlist.infile = evtfile+ccd+bkgparams
+    dmlist.infile = f"{evtfile}{ccd}{bkgparams}"
     dmlist.opt = "counts"
 
     obs_counts = float(dmlist())
 
     dmlist.punlearn()
-    dmlist.infile = bkgfile+ccd+bkgparams
+    dmlist.infile = f"{bkgfile}{ccd}{bkgparams}"
     dmlist.opt = "counts"
 
     bkg_counts = float(dmlist())
 
     return obs_counts/bkg_counts
```

Comment:
Not for this release, but I think it would be easier to uderstand if we just used crates and did something like (untested)

    obs_counts = read_file(f"{evtfile}{ccd}{bkgparams").get_nrows()

and something similar for `bkg_counts`

Reply:
👍I was actually considering this during the clean up but wasn't sure if it's a preferable approach...